src/main/java/command/stats/StatsProfitCommand.java

```diff
@@ -0,0 +1,18 @@
+package command.stats;
+
+import stats.OrderStatistics;
+import ui.Parser;
+
+public class StatsProfitCommand implements StatsCommand {
+    /**
+     * Prints the profit in the order statistics
+     *
+     * @param stats OrderStatistics object containing the statistics
+     * @param inputText the input text that was entered by the user
+     */
+    public static void execute(OrderStatistics stats, String inputText) {
+        String[] indexString = Parser.splitInput(Parser.analyzeInput(inputText), inputText);
+        double cost = Double.parseDouble(indexString[0]);
```

Comment:
Does this need to be in a try-catch block in case parseDouble() throws an exception?